nuget/Lykke.СryptoFacilities/Models/Common/Side.cs

```diff
@@ -0,0 +1,17 @@
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Lykke.СryptoFacilities.Models.Common
+{
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum Side
```

Comment:
Add default value to None (unspecified) to avoid errors while requesting with empty value.